go.mod

```diff
@@ -72,6 +72,7 @@ require (
 	k8s.io/kubelet v0.23.0
 	k8s.io/kubernetes v1.23.0
 	k8s.io/legacy-cloud-providers v0.23.0
+	k8s.io/pod-security-admission v0.0.0
```

Comment:
That's an odd version. Should be at least v0.23.0.

Reply:
indeed, this is what `go mod tidy` generated. let me try to fix this manually.